arbitrum-docs/arbos/l1-to-l2-messaging.mdx

```diff
@@ -1,64 +1,86 @@
 
-In most use cases, the submitter will provide gas and will intend for the immediate redeem to succeed, with later retries available only as a backup mechanism should the immediate redeem fail. (It might fail, for example, because the L2 gas price has increased unexpectedly.) In this way, an L1 contract can submit a transaction to L2 in such a way that the transaction will normally run immediately at L2 but allowing any party to retry the transaction should it fail.
 
-When a Retryable is redeemed, it will execute with the sender, destination, callvalue, and calldata of the original submission. The callvalue will have been escrowed during the initial submission of the Retryable, for this purpose. If a Retryable with callvalue is eventually discarded, having never successfully run, the escrowed callvalue will be paid out to a "beneficiary" account that is specified in the initial submission.
+Note that during redemption of a ticket, attempts to cancel the same ticket, or to schedule another redeem of the same ticket, will revert. In this manner retryable tickets are not self-modifying.
 
-A Retryable's beneficiary has the unique power to [`cancel`](./precompiles#ArbRetryableTx) the Retryable. This has the same effect as the Retryable timing out.
 
-[moved_link]: https://github.com/OffchainLabs/nitro/blob/fa36a0f138b8a7e684194f9840315d80c390f324/arbos/tx_processor.go#L191
+If a ticket with a callvalue is eventually discarded, having never successfully run, the escrowed callvalue will be paid out to a `callValueRefundAddress` account that was specified in the initial submission (step 1).
 
-### Redeeming a Retryable
+:::note Important Notes:
 
-If a redeem is not done at submission or the submission's initial redeem fails, anyone can attempt to redeem the retryable again by calling [`ArbRetryableTx`](precompiles#ArbRetryableTx)'s [`redeem`](./precompiles#ArbRetryableTx) precompile method, which donates the call's gas to the next attempt. ArbOS will [enqueue the redeem][enqueue_link], which is its own special `ArbitrumRetryTx` type, to its list of redeems that ArbOS [guarantees to exhaust][exhaust_link] before moving on to the next non-redeem transaction in the block its forming. In this manner redeems are scheduled to happen as soon as possible, and will always be in the same block as the tx that scheduled it. Note that the redeem attempt's gas comes from the call to [`redeem`](./precompiles#ArbRetryableTx), so there's no chance the block's gas limit is reached before execution.
+- One can redeem live tickets using the [Arbitrum Retryables Transaction Panel][retryable_dashboard_link]
+- The calldata of a ticket is saved on L2 until it is redeemed or expired 
+- Redeeming cost of a ticket will not increase over time, it only depends on the size of the calldata being kept on L2
+- The reason a ticket expires after one week is so it frees up the space it is no longer using after that period
```

Comment:
this feels like it belongs elsewhere (since it's design rationale, which doesn't feel like "important note" IMO)

Reply:
We had some teams asking us about why tickets expire after all? And that's the only reason I squeezed it here. Let's remove it though, don't like it here either